Wizardry/Core/BattleSys/Source/PreBattleCalc.c

```diff
@@ -1207,6 +1207,15 @@ void PreBattleCalcSkills(struct BattleUnit * attacker, struct BattleUnit * defen
                 attacker->battleSpeed += SKILL_EFF0(SID_AirRaidSpeed);
             break;
 #endif
+
+#if (defined(SID_SteadyBrawler) && (COMMON_SKILL_VALID(SID_SteadyBrawler)))
+        case SID_SteadyBrawler:
+            if (attacker->battleSpeed - defender->battleSpeed >= BATTLE_FOLLOWUP_SPEED_THRESHOLD)
```

Comment:
just as what I have told， it is bad to only check on speed

Reply:
Then do I move the check to `battleOrder.c` and see if a fighter related skill has been applied first?